recipes/iDQ/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "iDQ" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "31efed79f9a0e0eb03ef174f5f8e48d3d23a3213dd95616a89730661fb385669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k or win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - numpy >=1.7.1
+    - scipy
+    - ligo-common
+    - ligo-segments
+    - numpy>=1.7.1
```

Comment:
There are duplicated dependencies that should be removed (`numpy`, `ligo-common`, `ligo-segments`, possibly others).

Reply:
thanks for catching this, I removed this in the latest commit